build.gradle

```diff
@@ -236,6 +236,10 @@ dependencyManagement {
       entry 'tomcat-embed-websocket'
     }
 
+    dependencySet(group: 'com.fasterxml.jackson.core', version: '2.9.10') {
+      entry 'jackson-databind'
+    }
+
```

Comment:
Then there is no need to include `testCompile` ;)

Reply:
testCompile where @doncem ?

Reply:
we were trying to fix the security vulnerabilities. 
> One or more dependencies were identified with known vulnerabilities in sscs-bulk-scan:

>gradle-pitest-plugin-1.4.5.jar (cpe:2.3:a:gradle:gradle:1.4.5:*:*:*:*:*:*:*) : CVE-2019-11065, CVE-2019-15052, CVE-2019-16370
>wiremock-standalone-2.20.0.jar/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml (pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.8.11.2, cpe:2.3:a:fasterxml:jackson:2.8.11.2:*:*:*:*:*:*:*, cpe:2.3:a:fasterxml:jackson-databind:2.8.11.2:*:*:*:*:*:*:*) : CVE-2019-14540, CVE-2019-16335 

We've upgraded pitest plugin to the latest and forced to use the latest jackson-databind. Although v 2.8.11.2 is not pulled in, `dependencyCheckAnalyze` still complains.

Reply:
I'm afraid you cannot bump that for wiremock :(. Will have to be suppressed. Or try use `2.25.0` I believe it is all good now with wiremock on that version. Unless I'm mixing with something else nowadays.